Basic/Complex/complex.pd

```diff
@@ -25,6 +25,34 @@ pp_addpm {At => Top}, <<'EOD';
 
 PDL::Complex - handle complex numbers
 
+=head2 Native complex support
+
+There are important changes in complex number handling. Native C level support
+has been added, most functions and modules in the core distribution are supporting
+these as well. 
+
+PDL can now handle complex numbers natively as scalars. This has the advantage that
+real and complex valued piddles have the same dimensions. Consider this when writing
+code in the future.
+
+Important functions are:
+
+=for
+
+=item ci() - returns imaginary unit 
+
+=item creal, cimag, cabs, carg - return real, imaginary, absolute (length) and argument (angle) of 
+complex numbers.
+
+=item conj - complex conjugate
+
+Most exisitng functions work well regardless of input. Some require special handling, e.g. everything
```

Comment:
Spelling

Reply:
Also, are you sure that comparison is done only on the real part? If so, this is a bug and needs fixing. Can you add comments in the discussion of this PR showing it's true?